examples/official-storybook/config.js

```diff
@@ -60,8 +62,60 @@ addParameters({
   },
 });
 
-function importAll(req) {
-  req.keys().forEach(filename => req(filename));
+// TODO -- somehow don't use window for this cross HMR storage
+const { previousExports = {} } = window; // eslint-disable-line no-undef
```

Comment:
You can use module.hot.data:

https://webpack.js.org/api/hot-module-replacement/#dispose-or-adddisposehandler-

```
dispose (or addDisposeHandler) 
Add a handler which is executed when the current module code is replaced. This should be used to remove any persistent resource you have claimed or created. If you want to transfer state to the updated module, add it to given data parameter. This object will be available at module.hot.data after the update.

module.hot.dispose(data => {
  // Clean up and pass data to the updated module...
});
```

Reply:
OK, that worked, thanks @ndelangen